staging/src/k8s.io/pod-security-admission/test/testdata/baseline/v1.0/fail/hostnamespaces0.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: hostnamespaces0
+spec:
+  containers:
+  - image: k8s.gcr.io/pause
+    name: container1
+    securityContext: {}
+  hostIPC: true
+  initContainers:
+  - image: k8s.gcr.io/pause
+    name: initcontainer1
+    securityContext: {}
+  securityContext: {}
```

Comment:
looks like the fixtures need to be regenerated after dropping the `ensureSecurityContext` call